.github/workflows/labels.yml

```diff
@@ -23,3 +24,44 @@ jobs:
           dot: true
           sync-labels: true
 
+  issue-linked:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: octokit/graphql-action@v2.x
+        name: Get PR data from GraphQL API
+        id: get_linked_issue
+        with:
+          query: |
+            query issue($owner:String!,$repo:String!,$pr:Int!) {
```

Comment:
Love these improvements!!

When we manually add the label, we also check the author of the PR is who the issue is assigned to also, is that already done or is that something we can do?

Reply:
I will look int it

Reply:
@eddiejaoude The updated code handles the assignee check